include/maxscale/protocol/mysql.h

```diff
@@ -353,6 +360,8 @@ typedef struct
     int                    ignore_replies;               /*< How many replies should be discarded */
     GWBUF*                 stored_query;                 /*< Temporarily stored queries */
     bool                   collect_result;               /*< Collect the next result set as one buffer */
+    wait_gtid_state_t      wait_gtid_state;              /*< Determine boundray of wait gtid result and client query result */
```

Comment:
These can be moved into readwritesplit as it is the only module that uses it. The GTID tracking is useful for other modules as well so it is good to keep in the protocol module.